third_party/vulkan/loader/loader.c

```diff
@@ -3789,8 +3789,7 @@ VkResult loader_create_instance_chain(const VkInstanceCreateInfo *pCreateInfo,
                 continue;
             if ((fpGIPA = layer_prop->functions.get_instance_proc_addr) ==
                 NULL) {
-                if (layer_prop->functions.str_gipa == NULL ||
-                    strlen(layer_prop->functions.str_gipa) == 0) {
+                if (strlen(layer_prop->functions.str_gipa) == 0) {
```

Comment:
Correct me if I'm wrong, but `layer_prop->functions.str_gipa` is an array of fixed size allocated as a part of `functions` struct — so if `functions` is not NULL, str_gipa isn't either.